common/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM ghcr.io/dependabot/dependabot-updater-core
```

Comment:
Might want to add a comment saying "placeholder docker file used only for running unit tests here in common"

otherwise it'll be less clear to anyone who randomly discovers this file while code spelunking...